tests/test_dataclasses.py

```diff
@@ -380,3 +380,28 @@ class Outer:
             }
         },
     }
+
+
+def test_initvar():
+    from math import pi
+
+    InitVar = dataclasses.InitVar
+
+    @pydantic.dataclasses.dataclass
+    class RadiansWithInitVar:
+        rad: float = dataclasses.field(init=False)
+        deg: InitVar
+
+        def __post_init__(self, deg):
+            if deg:
+                self.rad = float(deg) / 180 * pi
+
+        @pydantic.validator("rad")
+        def validate_radians(cls, rad):
```

Comment:
is this required to test `InitVar`? If not please remove or move to another test.